src/styles/main.scss

```diff
@@ -1,3 +1,35 @@
+$colors:
+  1 rgb(100%, 0%, 0%),
+  2 rgb(80%, 0%, 0%),
+  3 rgb(60%, 0%, 0%),
+  4 rgb(40%, 0%, 0%),
+  5 rgb(20%, 0%, 0%),
+  6 rgb(0%, 0%, 0%);
+
 body {
   margin: 0;
 }
+
+.blocks {
+  max-width: 1199px;
```

Comment:
remove max-width, it should be expanded to the whole width of the window
<img width="1438" alt="Screenshot 2022-05-04 at 15 37 21" src="https://user-images.githubusercontent.com/57598825/166682521-12237a53-1767-4455-ab9f-73a2dc71487c.png">